Makefile

```diff
@@ -22,3 +22,9 @@ test: fmt
 			./test/integ-test/integ_test.py
 
 .PHONY: all clean docs
+
+binaries: linux windows
+linux:
+		GOOS=linux ${GO} build -o bin/shoelaces -ldflags "-s -w"
+windows:
+		GOOS=windows ${GO} build -o bin/shoelaces.exe -ldflags "-s -w"
```

Comment:
Given that we're repeating the flags, can you put them in a variable?
```
LDFLAGS += -s -w
```

and then you can use them like
```
-ldflags '$(LDFLAGS)'
```